Readme.md

```diff
@@ -113,6 +114,78 @@ $ make test
   - [Julian Gruber](https://github.com/juliangruber)
   - [Yiyu He](https://github.com/dead-horse)
 
+
+## Backers
+
+Support us with a monthly donation and help us continue our activities. [[Become a backer](https://opencollective.com/koajs#backer)]
```

Comment:
The first backer's badge has been changed to a Become a Backer icon: https://opencollective.com/static/images/become_backer.svg

That makes this link (at the end of the line) repetitive.